src/floating-focus.scss

```diff
@@ -1,24 +1,19 @@
 .floating-focus {
-	$border-size: 2px;
-
-	border: $border-size solid currentColor;
-	border-radius: 4px;
-	position: fixed;
-	top: 50%;
-	left: 50%;
+	border: 0 solid currentColor;
+	position: absolute;
 	transform: translate(-50%, -50%);
 	opacity: 0;
 
-	will-change: top, left;
+	will-change: top, left, width, height;
```

Comment:
From [this documentation](https://developer.mozilla.org/en-US/docs/Web/CSS/will-change) it seems that width and height aren't valid values for this?

Reply:
Ah I see, guess I didn't check that doc page thoroughly enough.

And I suppose you refrained from also adding `border-width` and `border-radius` because they make such little difference for performance that any optimisation would be redundant?

Reply:
Yes, exactly. And because the browser forces itself to keep each property animation state in memory while the will-change property is set for that property.